### PR TITLE
EvaluatePostixExpression added in Stack

### PR DESCRIPTION
**Pull request by vishal-bhola**
Evaluation Of Postfix Expression is added in Stack folder.

### CODE REVIEW

File: Algorithms/Stack/EvaluatePostfixExpression.txt [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+// Given a postfix expression, the task is to evaluate the expression and print the final value. Operators will only include the basic // arithmetic operators like *,/,+ and - . 
+
+// Input format: The first line of input will contains an integer T denoting the no of test cases . Then T test cases follow. Each //	         test case contains an postfix expression.
+
+// Output format: For each test case, in a new line, evaluate the postfix expression and print the value.
+
+// Sample Input :
+// 2
+// 231*+9-
+// 123+*8-
```
Comment (Bhavnaharitsa) [inline Algorithms/Stack/EvaluatePostfixExpression.txt]: Use a single large comment for multiple lines than commenting every line

File: Algorithms/Stack/EvaluatePostfixExpression.txt [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+                return result;
+            default:
+                return 0;
+        }
+        
+    }
+    
+	public static void main (String[] args) 
+	{
+		Scanner scan = new Scanner(System.in);
```
Comment (Bhavnaharitsa) [inline Algorithms/Stack/EvaluatePostfixExpression.txt:45]: a more simpler approach could be used using stack...this takes more time complexity

File: Algorithms/Stack/EvaluatePostfixExpression.txt (hunk 1 above)
Comment (vishal-bhola) [reply Algorithms/Stack/EvaluatePostfixExpression.txt]: Comment section part is done.

File: Algorithms/Stack/EvaluatePostfixExpression.txt (hunk 2 above)
Comment (vishal-bhola) [reply Algorithms/Stack/EvaluatePostfixExpression.txt:45]: I think this is the simplest approach as it's time complexity O(n) for each test case.

**Comment by vishal-bhola** [comment]
Changes been made please review.

**Comment by Bhavnaharitsa** [comment]
Looks good!

**Comment by vishal-bhola** [comment]
Could you please update the points of this PR in leaderboard as well.

**Pull request merged**